NewsletterAdminBundle/Transformer/NewsletterSubscriberTransformer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace OpenOrchestra\NewsletterAdminBundle\Transformer;
+
+use OpenOrchestra\BaseApi\Facade\FacadeInterface;
+use OpenOrchestra\BaseApi\Transformer\AbstractTransformer;
+use OpenOrchestra\Newsletter\Model\NewsletterSubscriberInterface;
+use OpenOrchestra\NewsletterAdminBundle\Facade\NewsletterSubscriberFacade;
+
+/**
+ * Class NewsletterSubscriberTransformer
+ */
+class NewsletterSubscriberTransformer extends AbstractTransformer
+{
+    /**
+     * @param mixed|NewsletterSubscriberInterface $newsletterSubscriber
+     *
+     * @return FacadeInterface
+     */
+    public function transform($newsletterSubscriber)
```

Comment:
can you check if $newsletterSubscriber is an instance of NewsletterSubscriberInterface and throw an ``TransformerParameterTypeException()`` if it isn't like other transformer.